src/App.css

```diff
@@ -1,22 +1,272 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 .nav {
+  padding: 24px 4%;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
   display: flex;
-  border: 2px solid rgb(177, 172, 172);
+  align-items: center;
+  grid-gap: 40px;
+  gap: 40px;
 }
 
-.navleft {
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
-  margin-left: 5%;
+.navtitle {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
 }
 
 .navlist {
   display: flex;
-  gap: 20%;
   list-style: none;
+  flex-grow: 1;
+  list-style-type: none;
+  grid-gap: 40px;
+  gap: 40px;
 }
 
-.listLink {
+.listLink1,
+.listLink2 {
   text-decoration: none;
-  color: black;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  color: #121212;
+  letter-spacing: 1.9px;
+}
+
+.listLink2 {
+  opacity: 0.5;
+}
+
+.navbutton {
+  cursor: pointer;
+  color: #0290ff;
+  width: 2.813rem;
+  height: 2.813rem;
+  border-radius: 50%;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+}
+
+.container {
+  padding: 4%;
+}
+
+.booklist {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  grid-gap: 0.938rem;
+  gap: 0.938rem;
+}
```

Comment:
- [ ] You did an excellent job working on styling your app! 💯 However,  I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓